frontend/pendingTranslations.ftl

```diff
@@ -0,0 +1,96 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
These are all new strings I had to add, some of them from places that were untranslated in the old site, some of them from places that just didn't have strings in the old site (mostly alt attributes and such), and some for features that just weren't present in the old site yet.

The strings here will show up until the same string IDs are present in the l10n repo. No more `???`s!